server/vulnerabilities/cve_test.go

```diff
@@ -148,7 +148,7 @@ func TestSyncsCVEFromURL(t *testing.T) {
 
 	tempDir := t.TempDir()
 	err := SyncCVEData(
-		tempDir, config.FleetConfig{Vulnerabilities: config.VulnerabilitiesConfig{CVEFeedPrefixURL: ts.URL}})
+		tempDir, config.FleetConfig{Vulnerabilities: config.VulnerabilitiesConfig{CVEFeedPrefixURL: ts.URL + "/feeds/json/cve/1.1/"}})
```

Comment:
Have the `httptest.Server` created in line 138 check that the request path is the one set here?